zbus/src/proxy/mod.rs

```diff
@@ -25,6 +24,10 @@ use crate::{
     AsyncDrop, Connection, Error, Executor, MatchRule, MessageStream, OwnedMatchRule, Result, Task,
 };
 
+use crate::abstractions::logging::{debug, trace};
+#[cfg(feature = "tracing")]
+use tracing::Instrument;
```

Comment:
Best if we just remove the import and use it with full path. Best to minimize the conditional code.